packages/tizen_bundle/README.md

```diff
@@ -0,0 +1,133 @@
+# tizen_bundle
+
+Tizen bundle APIs.
+
+## Usage
+
+To use this package, add `tizen_bundle` as a dependency in your `pubspec.yaml` file.
+
+```yaml
+depenedencies:
+  tizen_bundle:
+
+```
+
+## Adding Content to a Bundle
+
+The bundle content is in the from of key-value pairs, The key is always a string. The value can be of the following types:
```

Comment:
comma → dot?

Reply:
Done. The is changed the(lowercase).

Reply:
It's worse. You cannot connect two sentences simply with a comma. You may either insert "where" between the sentences or use a dot instead.

Reply:
Done.